C++/include/algorithm/sorting/README.md

```diff
@@ -0,0 +1,129 @@
+# Sorting
+Sorting algorithms are designed to arrange data in a particular order. Sorting algoritms have a few properties:
+- #### 1. Stable and Unstable Sorting:
```

Comment:
I think the formatting here would be improved with something like this:

```
#### Stable and unstable sorting
**Stable:** Identical elements remain in the same order as before sorting.  
**Unstable:** Identical elements may be sorted in any order.
```

Note the capitalization, colon placement, etc.